roles/openshift_health_checker/openshift_checks/mixins.py

```diff
@@ -2,17 +2,14 @@
 from openshift_checks import get_var
 
 
-class NotContainerized(object):
+class NotContainerizedMixIn(object):
```

Comment:
;) probably we should get rid of the mixins as you suggested originally, @detiber hehe

If we used class hierarchy:

```python
class PackageAvailability(NotContainerizedOpenShiftCheck):
    pass

class NotContainerizedOpenShiftCheck(OpenShiftCheck):
    pass
```

I still think, in this particular context, the suffix adds noise, but won't fight for it.